curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6553a44b1801991847d8cc69.md

```diff
@@ -0,0 +1,40 @@
+---
+id: 6553a44b1801991847d8cc69
+title: Step 44
+challengeType: 20
+dashedName: step-44
+---
+
+# --description--
+
+Next, modify your `print` call to print the encrypted message only and put it outside the `for` loop, so that the encrypted string is printed one time.
```

Comment:
```suggestion
Next, modify your `print()` call to print the encrypted message only and put it outside the `for` loop, so that the encrypted string is printed one time.
```